docs/source/tutorials/image.ipynb

```diff
@@ -318,40 +660,125 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from cleanlab import Datalab\n",
+    "lab.report(include_description=False)"
```

Comment:
why do you set `include_description=False` here? 

Remember this quickstart tutorial is somebody's very first experience ever seeing anything specific about how cleanlab is used!